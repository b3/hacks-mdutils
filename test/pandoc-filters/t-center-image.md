![tux](tux.svg){ .center }\ 

![tux](tux.svg)\ 

