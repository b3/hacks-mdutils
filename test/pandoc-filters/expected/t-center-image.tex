\centerline{\includegraphics{tux.svg}}~

\includegraphics{tux.svg}~
